src/pages/actors.js

```
@@ -0,0 +1,55 @@
+import React, { useEffect, useState } from "react";
+import { getActors } from "../services/actorServices";
+import ActorCard from "../components/actor_card";
+import Loading from "../components/loading";
+import Error from "../components/error";
```

Biome

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)